contracts/src/v0.8/functions/dev/v1_X/ocr/OCR2Base.sol

```diff
@@ -320,8 +316,10 @@ abstract contract OCR2Base is ConfirmedOwner, OCR2Abstract {
       emit Transmitted(configDigest, uint32(epochAndRound >> 8));
 
       ConfigInfo memory configInfo = s_configInfo;
-      // solhint-disable-next-line custom-errors
-      require(configInfo.latestConfigDigest == configDigest, "configDigest mismatch");
+      // The following check is disabled to allow both current and proposed routes to submit reports using the same OCR config digest
+      // Replay attacks are not a risk to Chanlink Functions because
```

Comment:
Instead of calling out replay attacks you can make a more generic statement that request IDs are globally unique and validated in Coordinator.

Also typo "Chanlink".